CMakeLists.txt

```diff
@@ -12,6 +12,9 @@ endif (POLICY CMP0135)
 
 project(mender)
 
+include(cmake/asan.cmake)
+include(cmake/memsan.cmake)
+include(cmake/threadsan.cmake)
```

Comment:
I guess you took these from somewhere? Would be nice to mention it in the commit message. Also, what's the license?

Reply:
I adapted them from `learning CMake` the book. I'll mention it in the PR :+1: 

It still feels like this should be able to meta-program a bit though, like this is bacially a function rendering this template with two variables...

Reply:
Simplified it quite a bit now. Turns out CMake has functions :tada: